src/main/webapp/gameStage.html

```diff
@@ -13,8 +13,15 @@
   <script src="gameDialogue.js"></script>
 </head>
 <body onload = "loadFunctions()">
+    
+  <audio controls autoplay loop style="float:left;"> <!--hidden tag can be added-->
+    <source src="music/lofi-sound.mp3" type="audio/mpeg">
+    If you're reading this, audio isn't supported  in your current web browser. 
```

Comment:
I don't think this bit is super necessary. If the user is using an unsupportive browser, this string would follow them through the game, I think we can scrap it.